src/get_bert_activations.py

```diff
@@ -0,0 +1,90 @@
+import torch
+import numpy as np
+import pickle as pkl
+from tqdm import tqdm
+from transformers import BertTokenizer, BertModel
+from nltk.tokenize import sent_tokenize
+from pathlib import Path
+import os
+
+
+MAX_INPUT_SIZE = 512
+
+DIR = Path('../data/alice/sample3/')
+DOC_TO_READ = DIR / 'doc.txt'
+
+TOKENS_OUTPUT = DIR / 'tokens.pickle'
+ACTIVATIONS_OUTPUT = DIR / 'activations.npz'
+
+
+def tokenize_and_segment():
+    file = open(DOC_TO_READ)
+    lines = file.read().replace("\n", " ")
+    file.close()
+    
+    sents = sent_tokenize(lines)
```

Comment:
I had to go look up what 'sents' meant in the nltk library. Can we rename this var 'sentences'?